Description: Removes depenedency on windows specific components for Linux build
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: jenkins/core/pom.xml
===================================================================
--- jenkins.orig/core/pom.xml	2012-07-24 16:33:05.898057949 +0100
+++ jenkins/core/pom.xml	2012-07-24 16:33:06.810061201 +0100
@@ -720,32 +720,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>winsw</id>
-            <phase>generate-resources</phase>
-            <goals>
-              <!-- we use copy as this is a dependency from outside the reactor -->
-              <goal>copy</goal>
-            </goals>
-            <configuration>
-              <artifactItems>
-                <artifactItem>
-                  <groupId>com.sun.winsw</groupId>
-                  <artifactId>winsw</artifactId>
-                  <version>1.10</version>
-                  <classifier>bin</classifier>
-                  <type>exe</type>
-                  <outputDirectory>${project.build.outputDirectory}/windows-service</outputDirectory>
-                  <destFileName>jenkins.exe</destFileName>
-                </artifactItem>
-              </artifactItems>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
       <plugin><!-- set main class -->
         <artifactId>maven-jar-plugin</artifactId>
         <!-- version specified in grandparent pom -->
